packages/@adobe/spectrum-css-temp/components/badge/index.css

```
@@ -30,14 +30,42 @@ governing permissions and limitations under the License.
   -webkit-font-smoothing: subpixel-antialiased;
   -moz-osx-font-smoothing: auto;
   font-smoothing: subpixel-antialiased;
+
+  /* label should always have padding at the end */
+  .spectrum-Badge-label {
+    padding-inline-end: var(--spectrum-global-dimension-size-115);
+  }
+
+  .spectrum-Icon {
```

```suggestion
  .spectrum-Badge-icon {
```
we had a lot of issues when we first started using css-modules because this classname exists everywhere and it can't be shared across CSS modules. So lets name it in a way that indicates that more readily.